bookkeeper-server/src/main/java/org/apache/bookkeeper/net/BookieSocketAddress.java

```diff
@@ -81,7 +81,8 @@ public InetSocketAddress getSocketAddress() {
      * Maps the socketAddress to a "local" address.
      */
     public LocalAddress getLocalAddress() {
-        return new LocalAddress(socketAddress.toString());
+        // for local address, we just need "port" to differentiate different addresses.
+        return new LocalAddress("" + port);
```

Comment:
@sijie This change seems unrelated. Could you please tell why you need this change ?
I am fine with it, but I am curious as I am an user of LocalAddresses.

Reply:
because this change triggers different sequence on registering bookie address to LocalBookieRegistry, it fails NetworkLessTest. 

"localhost/127.0.0.1" vs "127.0.0.1". (after DNS resolution vs before DNS resolution)